web/src/components/Icons.js

```diff
@@ -1,23 +1,42 @@
 
 // to optimize bundle, explicitly importing only the icons used
-import faArrowRight from '@fortawesome/fontawesome-free-solid/faLongArrowAltRight';
-import faHelp from '@fortawesome/fontawesome-free-regular/faQuestionCircle';
-import faBell from '@fortawesome/fontawesome-free-solid/faBell';
-import faCheckCircle from '@fortawesome/fontawesome-free-solid/faCheckCircle';
-import faChevronDown from '@fortawesome/fontawesome-free-solid/faChevronDown';
-import faChevronUp from '@fortawesome/fontawesome-free-solid/faChevronUp';
-import faCircle from '@fortawesome/fontawesome-free-regular/faCircle';
-import faClock from '@fortawesome/fontawesome-free-regular/faClock';
-import faCode from '@fortawesome/fontawesome-free-solid/faCode';
-import faCog from '@fortawesome/fontawesome-free-solid/faCog';
-import faEdit from '@fortawesome/fontawesome-free-solid/faEdit';
-import faExclamationTriangle from '@fortawesome/fontawesome-free-solid/faExclamationTriangle';
-import faPlusCircle from '@fortawesome/fontawesome-free-solid/faPlusCircle';
-import faPrint from '@fortawesome/fontawesome-free-solid/faPrint';
-import faSpinner from '@fortawesome/fontawesome-free-solid/faSpinner';
-import faHelpSolid from '@fortawesome/fontawesome-free-solid/faQuestionCircle';
-import faSignOut from '@fortawesome/fontawesome-free-solid/faSignOutAlt';
+import {
+  faArrowRight,
+  faBell,
+  faCheckCircle,
+  faChevronDown,
+  faChevronUp,
+  faCode,
+  faCog,
+  faEdit,
+  faExclamationTriangle,
+  faHelpSolid,
+  faPlusCircle,
+  faPrint,
+  faSpinner,
+  faSignOut
+} from '@fortawesome/free-solid-svg-icons';
+
+import { faHelp, faCircle, faClock } from '@fortawesome/free-regular-svg-icons';
+
+// import faArrowRight from '@fortawesome/free-solid-svg-icons/faLongArrowAltRight';
+// import faHelp from '@fortawesome/free-regular-svg-icons/faQuestionCircle';
+// import faBell from '@fortawesome/free-solid-svg-icons/faBell';
+// import faCheckCircle from '@fortawesome/free-solid-svg-icons/faCheckCircle';
+// import faChevronDown from '@fortawesome/free-solid-svg-icons/faChevronDown';
+// import faChevronUp from '@fortawesome/free-solid-svg-icons/faChevronUp';
+// import faCircle from '@fortawesome/free-regular-svg-icons/faCircle';
+// import faClock from '@fortawesome/free-regular-svg-icons/faClock';
+// import faCode from '@fortawesome/free-solid-svg-icons/faCode';
+// import faCog from '@fortawesome/free-solid-svg-icons/faCog';
+// import faEdit from '@fortawesome/free-solid-svg-icons/faEdit';
+// import faExclamationTriangle from '@fortawesome/free-solid-svg-icons/faExclamationTriangle';
+// import faPlusCircle from '@fortawesome/free-solid-svg-icons/faPlusCircle';
+// import faPrint from '@fortawesome/free-solid-svg-icons/faPrint';
+// import faSpinner from '@fortawesome/free-solid-svg-icons/faSpinner';
+// import faHelpSolid from '@fortawesome/free-solid-svg-icons/faQuestionCircle';
+// import faSignOut from '@fortawesome/free-solid-svg-icons/faSignOutAlt';
```

Comment:
What's goin' on with these suckers?

Reply:
Oops.  I commented them out as I replaced them, then forgot to delete them afterwards.  Will fix that...  😬